### ISSUE TITLE
Pillarbox sometimes not working

### ISSUE THREAD

**Issue by Jamerrone** [Aidoku/Aidoku#339]
This is probably because of the latest commit making the border cropping work in the webtoon reader.

https://github.com/Aidoku/Aidoku/assets/16142485/29b8c689-5fa5-4181-88b3-505a72c242ee

**Comment by Skittyblock** [comment]
I can't reproduce this; it's working perfectly fine for me.

**Comment by Jamerrone** [comment]
Yeah, not sure what to tell you. As you can see in the video, it was clearly broken, but like I mentioned yesterday on discord it does not appear to break consistently. I will keep an eye on it, you can close it if you want. I can always contact you in discord about it.

**Comment by Skittyblock** [comment]
yeah I'll leave this open since the video clearly shows it's an issue, but let me know if it happens again or if you figure out how it might be triggered

**Comment by Jamerrone** [comment]
@Skittyblock i still don't have any clear steps on how to reproduce this issue. I was able to record a version of this bug that fixed itself by scrolling, so it definitely is an issue.https://github.com/Aidoku/Aidoku/assets/16142485/8bda18f3-c57d-436e-83b8-8681c99ae20f

**Comment by Jamerrone** [comment]
@Skittyblock I'm now able to 100% reproduce this issue. It happens whenever I am in horizontal orientation and the Pillarbox orientation is set to "landscape". As long as I set it to "both" it works fine. So it's a minor bug, but it's an inconvenience because I read on a phone and I really need all the space I can get while reading vertically. I use it horizontally to get rid of the notch.

**Comment by Skittyblock** [comment]
I'm still unable to reproduce this.

**Comment by Jamerrone** [comment]
This does not seem to be an issue anymore.
